### ISSUE TITLE
Works on html but not xhtml - bug or my bad code?

### ISSUE THREAD

**Issue by kevlin79** [RobinHerbots/Inputmask#1771]
Have a strange issue and not sure it is a bug or bad code from me. The exact same code used in html or xhtml gives that the xhtml version doesn't work. Any thoughts appreciated.

Html code (xhtml further down)
<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN">

`<html>`
`<head>`
    `<title>Title</title>`
   ` <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js" type="text/javascript">`
`</script>`
   ` <script src="inputmask18/inputmask.js" type="text/javascript">`
`</script><!-- mask -->`

  `  <script src="inputmask18/jquery.inputmask.js" type="text/javascript">`
`</script><!-- mask -->`

   ` <script type="text/javascript">`
`$(document).ready(function(){`
	   ` $("input").inputmask("999 999999");  `               
`    }); `

 `   </script>`
`</head>`

`<body>`
   ` <div>`
       ` <input type="text" id="test" name="test" value="" />`
`    </div>`
`</body>`
`</html>`

Xhtml code
`<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">`
`<html xml:lang="en" xmlns="http://www.w3.org/1999/xhtml" lang="en">`
`<head>`
`    <title>Title</title>`
`    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js" type="text/javascript">
``</script>`
   ` <script src="inputmask18/inputmask.js" type="text/javascript">`
`</script><!-- mask -->`

  `  <script src="inputmask18/jquery.inputmask.js" type="text/javascript">`
`</script><!-- mask -->`

    `<script type="text/javascript">`
`$(document).ready(function(){`
	    `$("input").inputmask("999 999999");    `             
   ` }); `

 `   </script>`
`</head>`

`<body>`
   ` <div>`
      `  <input type="text" id="test" name="test" value="" />`
    `</div>`
`</body>`
`</html>`
